utils/ccloud_library.sh

```diff
@@ -382,6 +382,14 @@ function ccloud::create_service_account() {
   return 0
 }
 
+function ccloud:get_service_account_from_current_cluster_name() {
+  SERVICE_ACCOUNT_ID=$(confluent kafka cluster list -o json | jq -r '.[0].name' | awk -F'-' '{print $4 "-" $5;}')
+
+  echo $SERVICE_ACCOUNT_ID
```

Comment:
Do we want to print this? Or was this used for debugging?

Reply:
Yes, this is how the service account ID is passed back up to the calling script.